code/game/objects/items/weapons/landmines.dm

```diff
@@ -171,10 +243,164 @@
 
 	qdel(src)
 
+/**
+ * # EMP Landmine
+ *
+ * A landmine that emits an EMP pulse
+ */
 /obj/item/landmine/emp
 	icon_state = "emplandmine"
 
-/obj/item/landmine/emp/trigger(mob/living/L)
+/obj/item/landmine/emp/trigger(mob/living/triggerer)
 	spark(src, 3, alldirs)
 	empulse(src.loc, 2, 4)
 	qdel(src)
+
+
+/**
+ * # Phoron Landmine
+ *
+ * A landmine that only explodes when pressure is released
+ */
+/obj/item/landmine/standstill
+	name = "Standstill Landmine"
+	desc = "A landmine that only triggers if you release the pressure from the trigger."
+	icon_state = "standstill"
+	var/engaged_by = null
+
+/obj/item/landmine/standstill/Destroy()
+	STOP_PROCESSING(SSfast_process, src)
+	engaged_by = null
+	. = ..()
+
+/obj/item/landmine/standstill/trigger(mob/living/triggerer)
+	if(!engaged_by && !deactivated)
+		to_chat(triggerer, SPAN_HIGHDANGER("Something clicks below your feet, a sense of dread permeates your skin, better not move..."))
```

Comment:
You should probably give the mob some pain and a stun to make it more obvious, otherwise people will just keep walking through in case it's placed under a door or somewhere more conspicuous.

Reply:
They cannot keep walking, the proc shows an alert (tgui_alert) that prevents you from moving until you acknowledge it or re-focus the client window, it would also not make much sense that it gives you pain since it's not exploded yet